src/bindings/python/_flux/Makefile.am

```diff
@@ -173,15 +199,25 @@ lib-copy-vpath: ${fluxpyso_PYTHON} ${fluxpyso_LTLIBRARIES}
 		$(LN_S) .libs/$$LIB ./ $(STDERR_DEVNULL) || true; \
 	done
 
-all-local: lib-copy-vpath
+all-local: lib-copy-vpath ${fluxpyso_CODEGENJSON} ${fluxpyso_WRAPPERS}
+
+${fluxpyso_CODEGENJSON}:
+	$(AM_V_GEN)$(PYTHON) $(top_builddir)/src/bindings/python/codegen.py dump --out $(srcdir)/codegen ${fluxpyso_CODEGENINPUT}
+
+${fluxpyso_WRAPPERS}: ${fluxpyso_CODEGENJSON}
+	$(AM_V_GEN)$(PYTHON) $(top_builddir)/src/bindings/python/codegen.py generate --out $(top_srcdir)/src/bindings/python/flux/wrapper ${fluxpyso_CODEGENJSON}
```

Comment:
Crud, sorry, I made a mistake on this, it should be `top_srcdir` rather than `top_builddir`.

Reply:
haha yeah was just about to comment! Looks like we made a franken-path:
```
home/runner/work/flux-core/flux-core/src/bindings/python/_flux/../../../../src/bindings/python/codegen.py'
```